astroplan/core.py

```diff
@@ -1342,14 +1343,14 @@ def __init__(self, low, high):
         raise NotImplementedError
 
 
-class AboveAirmass(Constraint):
+class AirmassRange(Constraint):
     """
-    An object containing an airmass lower limit.
+    Constrain the airmass within a given range.
     """
 
-    def __init__(self, low):
+    def __init__(self, upper_limit, lower_limit=1):
```

Comment:
@eteq The problem with making `AirmassConstraint(2)` mean what you want is that it means we have to switch min / max order, i.e. have it be `AirmassConstraint(max, min)`, where all other constraints are `SomeConstraint(min, max)`.
So that can also be confusing to users.

But actually: I'll probably never use this constraint (we use Altitudes), so whatever you agree on: fine with me.